plugins/javadoc/src/test/kotlin/org/jetbrains/dokka/javadoc/JavadocModuleTemplateMapTest.kt

```diff
@@ -77,4 +78,65 @@ internal class JavadocModuleTemplateMapTest : AbstractJavadocTemplateMapTest() {
             assertEquals(DRI("com.test.package1"), assertIsInstance<RowJavadocListEntry>(list[1]).link.dri.single())
         }
     }
+
+    @Test
+    fun `single class with package documentation (kotlin)`() {
+        testTemplateMapInline(
+            query =
+            """
+            /src/module.md
+            # Module module1
+            ABC
+            
+            /src/source0.kt
+            package com.test.package0
+            class Test
+            """,
+            configuration = config.copy(
+                sourceSets = config.sourceSets.map { sourceSet ->
+                    sourceSet.copy(
+                        includes = setOf(File("src/module.md").also {println(it.absolutePath)})
+                    )
+                },
+                moduleName = "module1"
+            )
+        ) {
+            val modulePage = singlePageOfType<JavadocModulePageNode>()
+
+            val map = modulePage.templateMap
+            assertEquals("<p>ABC</p>", map["subtitle"].toString().trim())
+        }
+    }
+
+    @Test
+    fun `single class with long package documentation (kotlin)`() {
+        testTemplateMapInline(
+            query =
+            """
+            /src/module.md
+            # Module module1
+            Aliquam rerum est vel. Molestiae eos expedita animi repudiandae sed commodi.
+            Omnis qui ducimus ut et perspiciatis sint.
+            
+            Veritatis nam eaque sequi laborum voluptas voluptate aut.
+            
+            /src/source0.kt
+            package com.test.package0
+            class Test
+            """,
+            configuration = config.copy(
+                sourceSets = config.sourceSets.map { sourceSet ->
+                    sourceSet.copy(
+                        includes = setOf(File("src/module.md").also {println(it.absolutePath)})
+                    )
+                },
+                moduleName = "module1"
+            )
+        ) {
+            val modulePage = singlePageOfType<JavadocModulePageNode>()
+
+            val map = modulePage.templateMap
+            assertEquals("<p>Aliquam rerum est vel. Molestiae eos expedita animi repudiandae sed commodi. Omnis qui ducimus ut et perspiciatis sint.</p><p>Veritatis nam eaque sequi laborum voluptas voluptate aut.</p>", map["subtitle"].toString().trim())
```

Comment:
Could multiline string literals be used in this case? Having extremely long lines adds annoying horizontal scrolling and is not very good for readbility

```suggestion
            val expectedText = """
                <p>Aliquam rerum est vel. Molestiae eos expedita animi repudiandae sed commodi. 
                Omnis qui ducimus ut et perspiciatis sint.</p>
                <p>Veritatis nam eaque sequi laborum voluptas voluptate aut.</p>
            """.trimIndent()
            
            assertEquals(expectedText, map["subtitle"].toString().trim())
```

Here and in other two places